themes/hugo-hero-theme/layouts/pages/aboutlayout.html

```diff
@@ -4,9 +4,77 @@
 
 {{ define "main" }}
 {{ partial "hero-image-fullscreen.html" (dict "background" .Params.heroBackground "heading" .Params.heroHeading "subheading" .Params.heroSubHeading "content" .)}}
+<head>
+  <style>
+    .text{
+      padding-left:280px;
+      padding-right:280px;
+      text-align: justify;
+    }
+    .title{
+      text-align:left;
+      padding-left:650px;
+      padding-bottom:30px;
+    }
+    
+    .img{
+      float:right;
+      clear: right;
+    }
 
-<div id="home" class="container pt-8 pb-8 pt-md-10 pb-md-10">
-  <div class="row justify-content-center">
+    .feature{
+      background-blend-mode:lighten;
+      
+    }
+
+    .d-title{
+      margin-top:20px;
+      padding-left:650px;
+      
+    }
+   
+  </style>
+</head>
+
+<body>
+  <br>
+  <br>
+
+  <div class="welcome">
+  <h1 class="title">Our Mission </h1>
```

Comment:
The title is off centre - can we just exclude it?

![image](https://user-images.githubusercontent.com/43160110/84451932-64e7cd80-ac22-11ea-957c-442f2ced9b5b.png)

Reply:
I decided to put the title " our Mission" as the header of our page.

Reply:
Lit 🎈